tests/inspect.bats

```diff
@@ -115,9 +115,9 @@ load helpers
 }
 
 @test "inspect-format-docker-variant" {
-	# github.com/containerd/containerd/platforms.Normalize() converts Arch:"armhf" to Arch:"arm"+Variant:"v7",
+	# libimage.Normalize() converts Arch:"armhf" to Arch:"arm",
 	# so check that platform normalization happens at least for that one
-	run_buildah from --quiet --pull=false $WITH_POLICY_JSON --arch=armhf scratch
+	run_buildah from --quiet --pull=false $WITH_POLICY_JSON --arch=armhf --variant=v7 scratch
```

Comment:
What happens now if `--variant` _isn't_ specified?

Reply:
The `--variant` is not set, since internally `libimage.NormalizePlatform` does not calls `platform.Normalize`.  But if we want to retain the original behavior I think `libimage.NormalizePlatform` must be fixed.

Reply:
Same goes for `docker` i think if `variant` is not specified it is not set, consider the case of
 
`Docker build`: If variant is not specified it is not set
* `sudo docker build --no-cache --platform linux/armhf -t test .`

`buildkit`: Behaves same as current buildah behavior, sets variant `v7` even if variant was not specified
* `sudo docker buildx --no-cache --platform linux/armhf -t test .`

Reply:
I created a tryout PR in `c/common` https://github.com/containers/common/pull/1164 which makes `libimage.NormalizePlatform` work as it works for `buildkit` and also sets valid variant when it is explicitly specified as in case of `freebsd/arm64/v8`.

So following change is not needed.

Reply:
Although this difference between `buildkit` and `docker` looks really confusing to me plus buildkit behavior has a bug that it ignores the variant `v8` even it is set explicitly for certain cases like `--platform freebsd/arm64/v8`.

Reply:
Resolving this since this is fixed in c/common.